CI/travis.osx.after_success.sh

```diff
@@ -76,7 +76,11 @@ if [ "${DEPLOY}" = "deploy" ]; then
     fi
 
     if [ ! -z "$CERT_PW" ]; then
-      codesign --deep -s "$IDENTITY" "${HOME}/Desktop/Mudlet.dmg"
+      if [ "${public_test_build}" == "true" ]; then
+        codesign --deep -s "$IDENTITY" "${HOME}/Desktop/Mudlet PTB.dmg"
```

Comment:
do we want a space here, or would _ serve better?

Reply:
`Mudlet PTB.dmg` is already in use by the scripts elsewhere. I think being more human-friendly is better, but I don't mind too much.

Reply:
I'm not against it either way, just thought it worth touching on briefly.